spec/lib/reports/suppliers/pay_your_suppliers_report_spec.rb

```diff
@@ -49,12 +49,29 @@
 
   context "with taxes and fees" do
     let(:line_item) { order.line_items.first }
+    let!(:enterprise_fee) do
+      create(:enterprise_fee, enterprise: supplier, inherits_tax_category: true, fee_type: 'sales')
+    end
     let!(:fees_adjustment) do
-      create(:adjustment, originator_type: "EnterpriseFee", adjustable: line_item, amount: 0.1)
+      create(
+        :adjustment,
+        originator: enterprise_fee,
+        adjustable: line_item,
+        amount: 0.1,
+        label: "#{enterprise_fee.name} fee by supplier #{supplier.name}",
```

Comment:
Label are generated here : 
https://github.com/openfoodfoundation/openfoodnetwork/blob/5d9bb9a8d5ac2b491bc95d6fae1bab24c4f8c7de/lib/open_food_network/enterprise_fee_applicator.rb#L33-L36
Could you try to generate the test data using the `EnterpriseFeeCalculator` ? so the test won't break if we ever change the label ? see :
https://github.com/openfoodfoundation/openfoodnetwork/blob/5d9bb9a8d5ac2b491bc95d6fae1bab24c4f8c7de/lib/open_food_network/enterprise_fee_calculator.rb#L49-L55

That said we shouldn't be matching on the label see my earlier comment, so this is probably a moot point :smile:

Reply:
It has been addressed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12879/commits/ea270e3ab5d596fcf57f8607d2fc72d9463c202a